build/scripts/meta.yaml.schema

```diff
@@ -120,6 +120,18 @@
                 "type": "string",
                 "pattern": "[1-9][0-9]*([Gg]i|(Mm)i|(Kk)i|b)?"
               },
+              "memoryRequest": {
+                 "type": "string",
+                 "pattern": "[1-9][0-9]*([Gg]i|(Mm)i|(Kk)i|b)?"
```

Comment:
This regexp doesn't seem to be accurate (but I see it's copied from the limit above). This will match any number not starting with 0, and ignore the suffix completely in all cases except `Gi`.

Reply:
FIxed it